src/Schema/Tables/Symptoms.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[Symptoms]
+(
+	[Id] INT NOT NULL PRIMARY KEY IDENTITY, 
```

Comment:
In your previous table you use PatientAllergyId, AllergyId etc. And here you decide to use just Id. May be we will develop a common strategy in this question and will stick to it?